bedrock/firefox/templates/firefox/pocket.html

```diff
@@ -42,13 +42,13 @@
       </h2>
 
       {# Has toolbar button #}
-      <p class="main-cta" id="cta-try"><a href="https://support.mozilla.org/kb/where-pocket-button-firefox" class="button-flat try-pocket">{{ _('Try it now') }}</a></p>
+      <p class="main-cta" id="cta-try"><a href="https://support.mozilla.org/kb/where-pocket-button-firefox" class="button dark try-pocket">{{ _('Try it now') }}</a></p>
 
       {# No toolbar button #}
-      <p class="main-cta" id="cta-learn"><a href="https://support.mozilla.org/kb/where-pocket-button-firefox" class="button-flat">{{ _('Learn more') }}</a></p>
+      <p class="main-cta" id="cta-learn"><a href="https://support.mozilla.org/kb/where-pocket-button-firefox" class="button dark">{{ _('Learn more') }}</a></p>
 
       {# Old Firefox or Not Firefox #}
-      <p class="main-cta" id="cta-download"><a href="{{ url('firefox.new') }}?scene=2" class="button-flat-green">{{ _('Download Firefox') }}</a></p>
+      <p class="main-cta" id="cta-download"><a href="{{ url('firefox.new') }}?scene=2" class="button green">{{ _('Download Firefox') }}</a></p>
```

Comment:
Note to self: any reason we can't make this a real download button, now we can pass in custom strings?